content/docs/authorization/deployment/_index.md

```diff
@@ -62,27 +66,30 @@ A Storage Administrator can execute the installer or rpm package as a root user
       "hostName": "DNS_host_name"
     }
     ```
+>__Note__:
+- `DNS_host_name` refers to the hostname of the system in which the CSM for Authorization server will be installed. This hostname can be found by running `nslookup <IP_address>`
+- There are a number of ways to create certificates. In a production environment, certificates are usually created and managed by an IT administrator. Otherwise, certificates can be created using OpenSSL.
 
-    In the above template, `DNS_host_name` refers to the hostname of the system in which the CSM for Authorization server will be installed. This hostname can be found by running the below command on the system:
+2. In order to configure secure grpc connectivity, an additional subdomain in the format `grpc.DNS_host_name` is also required. All traffic from `grpc.DNS_host_name` needs to be routed to `DNS_host_name` address, this can be configured by adding a new DNS entry for `grpc.DNS_host_name` or providing a temporary path in the systems `/etc/hosts` file. 
 
-    ```
-    nslookup <IP_address>
+    ```json
+      {
+        "hostName": 
```

Comment:
AFAIK, this is not a valid JSON. That syntax is for key:values pairs. For multiple values, it should be something like:
```json
{
  "hostName": [
    "DNS_host_name",
    "grpc.DNS_host_name"
  ]
}
```

Btw, is the key really `hostName` not `hostname` ? `hostName` feels weird.

Reply:
fixed syntax and updated `hostName `to `hostname`